addons/account/wizard/account_invoice_send.py

```diff
@@ -86,7 +86,7 @@ def _send_email(self):
         if self.is_email:
             self.composer_id.send_mail()
             if self.env.context.get('mark_invoice_as_sent'):
-                self.mapped('invoice_ids').write({'invoice_sent': True})
+                self.mapped('invoice_ids').sudo().write({'invoice_sent': True})
```

Comment:
please add a comment explaining the sudo

Reply:
and voilà

Reply:
and voila quoi?

Reply:
git push?

Reply:
And revoilà

Reply:
finally :-p